src/logic/operate.js

```diff
@@ -0,0 +1,30 @@
+import Big from 'big.js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  const firstNum = Big(numberOne);
+  const secondNum = Big(numberTwo);
+  let total = 0;
+
+  switch (operation) {
+    case '%':
+      total = ((secondNum / 100) * firstNum);
+      break;
+    case '÷':
+      total = (firstNum / secondNum);
+      break;
+    case '*':
+      total = (firstNum * secondNum);
+      break;
+    case '-':
+      total = (firstNum - secondNum);
+      break;
+    case '+':
+      total = (firstNum + secondNum);
```

Comment:
Job well done but you don't have to do the calculating yourself, that where the use of `big` comes in -> [big js library](https://github.com/favio41/big-js#readme)
example: case subtraction will be -> `total.minus(next)`

OR

http://mikemcl.github.io/big.js/#mod